client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.jsx

```diff
@@ -119,16 +120,30 @@ const DataFilesBreadcrumbs = ({
   const paths = [];
   const pathComps = [];
 
+  const systems = useSelector(
+    (state) => state.systems.storage.configuration.filter((s) => !s.hidden),
+    shallowEqual
+  );
+
+  const root = useSystemDisplayName({ scheme, system });
```

Comment:
I think to get around the issue of rendering the name for `Community` and `Public` systems, we could amend the `findSystemOrProjectDisplayName` function that is called by `useSystemDisplayName`, and expand `findSystemOrProjectDisplayName` to find the name of the system based on the scheme, which is unique for `community` and `public`.

To implement in a general sense, we could "try" to find the system based on scheme, seeing if the `find` returns exactly 1 result, and fall back to other methods if that fails

Reply:
Oh, just looked at #767 🤣 . Nicely done